docs/guides/advanced/link-to-code-source.md

```diff
@@ -0,0 +1,45 @@
+!!! warning "Experimental"
+    This feature is experimental and may change in the future.
+
+    Please give us feedback on [Slack][slack].
+
+We support linking to the source code on GitHub, or any other VCS provider.
+
+![Link to GitHub](../../images/guide/link-to-github.gif)
+
+## Usage
+
+To enable source code linking with Logfire, you need to configure your project. Here's how:
+
+```python
+import logfire
+
+logfire.configure(
+    code_source=logfire.CodeSource(
+        repository='https://github.com/pydantic/logfire',
+        revision='main',
+        root_path='.',
```

Comment:
Should this just be the default?

Reply:
I can change this in a future PR, but I guess it makes sense to be the default yes.

Reply:
seems like it should be optional in the data class but the frontend needs to handle defaulting to it so that the env var isn't required

Reply:
frontend does handle default already